mono/metadata/threadpool-worker-default.c

```diff
@@ -761,13 +761,13 @@ monitor_thread (gpointer unused)
 		});
 
 		if (limit_worker_max_reached) {
-			mono_trace (G_LOG_LEVEL_INFO, MONO_TRACE_THREADPOOL, "[%p] monitor thread, limit_worker_max (%d) reached",
+			mono_trace (G_LOG_LEVEL_DEBUG, MONO_TRACE_THREADPOOL, "[%p] monitor thread, limit_worker_max (%d) reached",
```

Comment:
INFO on these?

Reply:
this one is kind of spammy, I think. I think it's probably ok at DEBUG